src/test/java/com/wikia/webdriver/testcases/renametool/RenameToolTests.java

```diff
@@ -0,0 +1,200 @@
+package com.wikia.webdriver.testcases.renametool;
+
+import com.wikia.webdriver.common.contentpatterns.PageContent;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.annotations.RelatedIssue;
+import com.wikia.webdriver.common.core.api.ArticleContent;
+import com.wikia.webdriver.common.core.api.UserRegistration;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.SignUpUser;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.properties.Credentials;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.minieditor.MiniEditorComponentObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.messagewall.MessageWall;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.renametool.ConfirmationModalPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.renametool.HelpPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.renametool.SpecialRenameUserPage;
+
+import org.joda.time.DateTime;
+import org.testng.annotations.Test;
+
+import java.time.LocalDate;
+
+@SuppressWarnings("SpellCheckingInspection")
+@InBrowser(browser = Browser.CHROME)
+@Execute(onWikia = "communitytest")
+@Test(groups = "renameTool")
+public class RenameToolTests extends NewTestTemplate {
+
+  @Test
+  @Execute(asUser = User.QARENAME)
+  public void UserProvidesCorrectNewNameDoesntClickUnderstandCheckbox() {
+    SpecialRenameUserPage renameUserPage = new SpecialRenameUserPage()
+        .open()
+        .fillFormData("ChesskyTest", "ChesskyTest", "q")
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "ChesskyTest" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)